pkg/kubectl/cmd/testing/fake.go

```diff
@@ -680,6 +685,7 @@ func testDynamicResources() []*discovery.APIGroupResources {
 			VersionedResources: map[string][]metav1.APIResource{
 				"v1beta1": {
 					{Name: "deployments", Namespaced: true, Kind: "Deployment"},
+					{Name: "thirdpartyresources", Namespaced: false, Kind: "ThirdPartyResource"},
```

Comment:
(Specifically, the tests mentioned in https://github.com/kubernetes/kubernetes/issues/40841#issue-204769102)